alfresco-solr-query-analytics/src/main/amp/config/alfresco/module/alfresco-solr-query-analytics/log4j.properties

```diff
@@ -1,13 +1,14 @@
 # SOLR query Logger
 # Define the file appender
-log4j.appender.solrQueryLog=org.apache.log4j.DailyRollingFileAppender
+log4j.appender.solrQueryLog=util.CustomLog4jAppender
 log4j.appender.solrQueryLog.File=/usr/local/tomcat/logs/solrquery.log
-log4j.appender.File.Append=true
-log4j.appender.File.DatePattern='.'yyyy-MM-dd
+log4j.appender.solrQueryLog.Append=true
+log4j.appender.solrQueryLog.encoding=UTF-8
 log4j.appender.solrQueryLog.layout=net.logstash.log4j.JSONEventLayoutV1
-log4j.appender.solrQueryLog.layout.ConversionPattern=%d{yyyy-MM-dd} %d{ABSOLUTE} %-5p [%c] [%t] %m%n
+log4j.appender.solrQueryLog.layout.ConversionPattern=%solrQueryLog{yyyy-MM-dd} %solrQueryLog{ABSOLUTE} %-5p [%c] [%t] %solrQueryLog%solrQueryLog
+log4j.appender.solrQueryLog.DatePattern='.'yyyy-MM-dd
+log4j.appender.solrQueryLog.MaxNumberOfDays=3
+log4j.appender.solrQueryLog.CompressBackups=true
 
-log4j.logger.org.alfresco.repo.search.impl.solr.SolrQueryHTTPClient=debug, solrQueryLog
```

Comment:
Weird that this was not already adjusted

Reply:
What do you mean?

Reply:
This was still pointing to the alfresco class, not our adjusted one. I expect it would not function with that configuration. However it was working, the solr query log lines were already routed to the correct external logfile.

Reply:
They were both there (eu.xenit and org.alfresco), that's why it worked.